app/components/modules/Transfer.jsx

```diff
@@ -75,9 +75,9 @@ class TransferForm extends Component {
                     ! values.to ? 'Required' : validate_account_name(values.to),
                 amount:
                     ! values.amount ? 'Required' :
-                    ! /^[0-9]*\.?[0-9]*/.test(values.amount) ? 'Amount is in the form 99999.999' :
+                    ! /^\d+(?:[\.\.]\d+)?$/.test(values.amount) ? 'Amount is in the form 99999.999' :
```

Comment:
They actually work identical in the validation after testing, but prefer this as it's cleaner.  Updated.